Lib9c/Action/IBuy5.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using Libplanet;
+using Libplanet.Action;
+
+namespace Nekoyume.Action
+{
+    /// <summary>
+    /// Common interface used before <see cref="IBuy5"/>.
```

Comment:
This comment seems unnatural. Hmm I'm not sure about my sense but how about:

```suggestion
    /// Common interface which has used since <see cref="Buy5"/>.
```